### ISSUE TITLE
[Sonic Generations] 

### ISSUE THREAD

**Issue by Escavalien** [thesupersonic16/HedgeModManager#180]
HMM Info:
```
    Version: 7.2-9
    Args: -gb hedgemmgens:https://gamebanana.com/mmdl/511145,Effect,6373
    StartDir: D:\Sonic Generations\
    Process Level: Administrator
    Game: Sonic Generations
    SteamGame: Sonic Generations
```

Exception:
```
    Type: NullReferenceException
    Message: Object reference not set to an instance of an object.
    Source: HedgeModManager
    Function: Boolean RequestItemData(GameBananaAPI.GBAPIItemDataBasic ByRef)
    StackTrace: 
       at GameBananaAPI.GBAPI.RequestItemData(GBAPIItemDataBasic& item)
       at GameBananaAPI.GBAPI.ParseCommandLine(String line)
       at HedgeModManager.HedgeApp.Main(String[] args)
    InnerException: 
```

**Comment by thesupersonic16** [comment]
Does this happen consistently? HMM doesn't seem to crash on my end.

**Comment by thesupersonic16** [comment]
Oh I see, one of the files on the page seems to cause the crash.

**Comment by thesupersonic16** [comment]
Just took a look, the issue was already fixed in dca73a1131485e5015d04bb5cc46b47616240071 which has not been released yet. The issue should be fixed in the next release of HedgeModManager.

**Comment by thesupersonic16** [comment]
HedgeModManager 7.2-10 has been released which should contain the fix for this issue. If the issue still occurs, please reopen this issue. Closing.
